Automattic/wp-calypso
# P2 signup: P2 Complete Profile

Addresses 4741-gh-automattic/p2.

In this PR, we add a new step to the P2 signup flow:

<img width="1498" alt="Screenshot on 2022-03-18 at 18-04-35" src="https://user-images.githubusercontent.com/4988512/159049554-6600fe04-5281-4aa5-9646-1199d03c6b44.png">

It allows uploading or modifying the user's Gravatar and changing their full name (`display_name`).

## Testing instructions

With an incognito window, navigate to `/start/p2`. Using a 10 minute email address, create a new user and verify the email. In the first tab, reload the signup flow (as the email confirmation link will take you to wordpress.com/production). After clicking on Continue, you should arrive to the Complete Profile step.

Try to upload a Gravatar and change your full name. The name must be longer than 3 characters and the Gravatar must be an image.
